cmake/FindBrotli.cmake

```diff
@@ -0,0 +1,45 @@
+#***************************************************************************
+#                                  _   _ ____  _
+#  Project                     ___| | | |  _ \| |
```

Comment:
Was this file taken from the CURL project? Or is this a a copy/paste artifact?

Reply:
Yes, taken from there, with some small modifications for letter case.